src/components/Footer/Footer.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <footer>
-    <p><b>Data Source:</b> <a href="https://www.iqair.com/us/air-pollution-data-api" target="_blank">IQAir Air Quality API</a></p>
-    <p><b>Built By:</b> Tashia Davis, Alia Peterson, and Allison Dietz. <a href="https://github.com/alia-peterson/ActivAir" target="_blank">View Repo on GitHub</a></p>
+    <p><b>Data Source:</b> <a href="https://www.iqair.com/us/air-pollution-data-api" target="_blank" rel="noreferrer">IQAir Air Quality API</a></p>
+    <p><b>Built By:</b> Tashia Davis, Alia Peterson, and Allison Dietz. <a href="https://github.com/alia-peterson/ActivAir" target="_blank" rel="noreferrer">View Repo on GitHub</a></p>
```

Comment:
Ohh what is this `rel="noreferrer"` ? I've never seen that before and google didn't really clear it up very much

Reply:
I thought it was more of an SEO thing? There is such a thing as linking out to too many external sites, which has SEO implications.

Reply:
It was highlighted by running the 'best practices' audit in Lighthouse, and Tashia's right, it's a security consideration as we incorporate external links